### ISSUE TITLE
maybe refresh images/check_env-2.png used in the readme

### ISSUE THREAD

**Issue by stonebig** [amueller/ml-workshop-1-of-4#1]
Requirements have change a bit since 2 years.
![image](https://user-images.githubusercontent.com/4312421/47933040-ab683880-ded3-11e8-87a5-796dd98c71cf.png)

**Comment by amueller** [comment]
Hey. You mean for the screen shot?

**Comment by stonebig** [comment]
Yes, so people don't get worried they don't have pydot...etc... when looking at github readme

**Comment by amueller** [comment]
Sorry on my phone I didn't see the title of the issue at first. Sure, do you want to create a PR for any/all of the repos? ;)

**Comment by stonebig** [comment]
no thanks.

**Comment by amueller** [comment]
fixed

**Comment by amueller** [comment]
actually no lol
